styles/linter-ui.less

```diff
@@ -17,64 +17,85 @@
   padding: 5px 5px 2px 5px;
   font-family: @font-family;
 
-  .message-with-severity(@color) {
+  .linter-message {
+    padding: 10px;
+
     color: #fff - @inset-panel-background-color; // invert;
     background: @app-background-color;
 
-    border-left: 8px solid @color;
-    border-right: 1px solid fade(contrast(@inset-panel-background-color), 5%);
-    &:first-child {
-      border-top: 1px solid fade(contrast(@inset-panel-background-color), 5%);
-    }
-    &:last-child {
-      border-bottom: 1px solid fade(contrast(@inset-panel-background-color), 5%);
-    }
+    .linter-excerpt {
+      padding: 5px;
```

Comment:
```suggestion
      padding: 5px;
      display: flex;
      align-items: center;
```

Reply:
This aligns all of the children of `.linter-exerpt` vertically.

If the message has multiple lines everything will still be centered. If we want everything centered with the first line of the message you can change this to `align-items: baseline;`.

Reply:
Now. The provider name takes its own column instead of being inline

![image](https://user-images.githubusercontent.com/16418197/106546279-8444ab00-64d0-11eb-854f-fc8fa48b5301.png)

vs

![image](https://user-images.githubusercontent.com/16418197/106408052-301fc500-6403-11eb-8bfa-7d6a16ad3084.png)

Reply:
`center`:

![image](https://user-images.githubusercontent.com/97994/106546649-4bf19c80-64d1-11eb-8ec4-c84ed647ec58.png)

`baseline`:

![image](https://user-images.githubusercontent.com/97994/106546656-4dbb6000-64d1-11eb-9966-4240adcfa5fa.png)

Reply:
We can also make the right buttons not wrap if we want to.

Reply:
Can we wrap the provider and message in another div? Then that wrapper would be the flex child that is aligned instead of the provider and message.

Reply:
Yes. we can. I will do it now.

Reply:
It looks better now. I updated the screenshots in the OP.

Reply:
![image](https://user-images.githubusercontent.com/97994/106549250-32068880-64d6-11eb-87e3-109f19a25b6b.png)